tests/integration/test.yml

```diff
@@ -17,7 +24,7 @@
 
   pre_tasks:
     - assert:
-        that: "lookup('dig', ansible_host) == lookup('dig', TEST_DOMAIN)"
+        that: "ansible_host == lookup('dig', TEST_DOMAIN)"
```

Comment:
The `ansible_host` isn't necessarily an IP address, but may be one.  So far `lookup('dig', ansible_host)` worked for both IP addresses and host names, and would always return the IP address, at least for the tests.  In other environments we had issues that it wouldn't work with IP addresses, but we never found out why.  So well, if this gets the tests passing, we will always have to use an IP address for `ansible_host` going forward, which is fine with me.

Reply:
Yeah, that's the thing. It started failing with NXDOMAIN.